src/main/kotlin/net/onelitefeather/clipboardconnect/services/SyncService.kt

```diff
@@ -96,6 +97,12 @@ class SyncService @Inject constructor(private val config: FileConfiguration, pri
             } catch (e: IOException) {
                 logger.error(MiniMessage.miniMessage().deserialize("<red>Failed to load redis.yml"), e)
                 plugin.server.pluginManager.disablePlugin(plugin)
+            } catch (e: RedisConnectionException) {
+                logger.error(MiniMessage.miniMessage().deserialize("<red>Failed to create a redis connection"), e)
+                plugin.server.pluginManager.disablePlugin(plugin)
+            } catch (e: Exception) {
+                logger.error(MiniMessage.miniMessage().deserialize("<red>Something went wrong to create a redis connection"), e)
```

Comment:
"Something went wrong while trying to initialize a redis connection"
Might sound better, but can be discussed what the end user (the reporter and the developer) might better understand